packages/jest-message-util/src/index.ts

```diff
@@ -331,6 +333,14 @@ export const formatResultsErrors = (
     .join('\n');
 };
 
+const removeBlankErrorLine = (str: string) =>
+  str
+    .split('\n')
+    // Lines saying just `Error:` are useless
+    .filter(line => !/^Error:?\s*/.test(line))
```

Comment:
Shouldn't this regex include `$` character at the end? I can imagine errors like `Error: Some message`.

Reply:
very much correct